explorer/src/style/main.css

```diff
@@ -67,3 +67,7 @@ svg {
     padding: 0 1.25rem;
   }
 }
+
+.general__badge {
```

Comment:
I think we should name this `general-badge`, or better yet `explorer-badge` because these properties are specific to the explorer.

Reply:
IMO, BEM styles should be concretely named in the sense that a _block_ should be something tangible i.e. a noun e.g. 'card' or 'badge' and then the element should be something within that _block_  (also a noun) like `heading` or `title` and then the modifier should be an adjective like  'active' or 'enabled'.

Reply:
Also, I think we should (as I don't see this as a concrete rule in BEM) never jump in and style a `.block__element` directly.  The `.block__element` styles should that cascade over the `.block` styles.